src/main/simulation.cpp

```diff
@@ -47,17 +47,33 @@ bool Dissolve::prepare()
     for (const auto &at : atomTypes())
         at->setIndex(count++);
 
+    // Store / update last-used pair potential cutoff
+    // If lastPairPotentialCutoff is nullopt, store the current value and move on leaving the cutoff to use as nullopt.
+    static std::optional<double> lastPairPotentialRange;
+    std::optional<double> newPairPotentialRange;
+    if (!lastPairPotentialRange)
+        lastPairPotentialRange = pairPotentialRange_;
+    else if (lastPairPotentialRange != pairPotentialRange_)
+    {
+        lastPairPotentialRange = pairPotentialRange_;
+        newPairPotentialRange = pairPotentialRange_;
+    }
+
     // Check Configurations
     std::set<const Species *> globalUsedSpecies;
     for (auto &cfg : configurations())
     {
+        // Regenerate cell array if the pair potential range has changed
+        if (newPairPotentialRange)
+            cfg->updateCells(7.0, newPairPotentialRange.value());
```

Comment:
There's a nice way to do this with the new monadic operations on `optional` in C++23, but that will have to wait.  Meanwhile, I thought that the above with clearer, but I'm open to disagreement.

```suggestion
        if (newPairPotentialRange)
            cfg->updateCells(7.0, *newPairPotentialRange.);
```

Reply:
Happy to dereference rather than `value()` - don't mind either approach.